lib/index.js

```diff
@@ -2388,22 +2403,7 @@ Stream.prototype.drop = function (n) {
     if (n <= 0) {
         return this;
     }
-    return this.consume(function (err, x, push, next) {
-        if (err) {
-            push(err);
-            next();
-        }
-        else if (x === nil) {
-            push(null, nil);
-        }
-        else {
-            n--;
-            if (n < 0) {
-                push(null, x);
-            }
-            next();
-        }
-    });
+    return this.slice(n);
```

Comment:
This is a bit pedantic, but can you change this to explicitly call `this.slice(n, Infinity)`?

We don't allow optional arguments in transforms since they all get auto-curried and placed in the top-level `_` object. So while `slice` does argument coercing, we don't want the code to suggest that the `end` argument optional.